src/data/nat/digits.lean

```diff
@@ -471,6 +472,22 @@ begin
   exact base_pow_length_digits_le' b m,
 end
 
+/-! ### Binary -/
+lemma base_two_digits_eq_bits (n : ℕ) : digits 2 n = n.bits.map (λ b, cond b 1 0) :=
```

Comment:
I'd just go for directly reflecting the syntax of the statement here.
```suggestion
lemma digits_two_eq_bits (n : ℕ) : digits 2 n = n.bits.map (cond b 1 0) :=
```
I was unsure about `cond` but it seems to be more useful than `if` here, since `b` is a boolean with a specific value.